source/interface/stat.js

```diff
@@ -47,5 +47,6 @@ function parseStat(result, filename, isDetailed = false) {
 }
 
 module.exports = {
-    getStat
+    getStat,
+    parseStat
```

Comment:
I think this is extra?

Reply:
I have used parseStat here https://github.com/perry-mitchell/webdav-client/pull/174/files/6c092e915c46ea0754eb498bca6696b10ff124d2#diff-f5ff98f224fcf64e04d02c119dfb0c33R37
Is it a problem exporting this function?